### PR TITLE
Add/examples

### PR DESCRIPTION
**Pull request by jakubczakon**
* added code examples
* added video tutorials
* added some blog posts showcasing neptune

### CODE REVIEW

File: docs/examples/code-examples.rst [hunk 1]
```diff
@@ -0,0 +1,276 @@
+.. _examples-code-examples:
+
+Code examples
```
Comment (kamil-kaczmarek) [inline docs/examples/code-examples.rst:3]: Entire thing is super-cool!

Comment (kamil-kaczmarek) [reply docs/examples/code-examples.rst:3]: nice idea to create it

File: docs/examples/video-tutorials.rst [hunk 2]
```diff
@@ -0,0 +1,232 @@
+.. _examples-neptune-video-tutorials:
+
+Video Tutorials
```
Comment (kamil-kaczmarek) [inline docs/examples/video-tutorials.rst:3]: There is quite a lot of videos - difficult to browse it / find something relevant.

Maybe some toc at the top?

**Review by kamil-kaczmarek — APPROVED** [review]

File: docs/examples/video-tutorials.rst (hunk 2 above)
Comment (jakubczakon) [reply docs/examples/video-tutorials.rst:3]: Figured I first want to see if people read it. If true then I would like to structure this thing more than just a toc

**Pull request merged**